project/AppDependencies.scala

```diff
@@ -2,31 +2,30 @@ import sbt.*
 
 object AppDependencies {
 
-  val hmrcMongoVersion = "1.7.0"
-  val bootstrapVersion = "7.23.0"
+  val hmrcMongoVersion = "1.8.0"
+  val bootstrapVersion = "8.5.0"
 
   val compile: Seq[ModuleID] = Seq(
     play.sbt.PlayImport.ws,
-    "uk.gov.hmrc"                  %% "play-frontend-hmrc-play-28"    % "8.5.0",
-    "uk.gov.hmrc"                  %% "play-conditional-form-mapping" % "1.13.0-play-28",
-    "uk.gov.hmrc"                  %% "bootstrap-frontend-play-28"    % bootstrapVersion,
-    "uk.gov.hmrc"                  %% "internal-auth-client-play-28"  % "1.10.0",
-    "uk.gov.hmrc.objectstore"      %% "object-store-client-play-28"   % "1.3.0",
-    "uk.gov.hmrc.mongo"            %% "hmrc-mongo-play-28"            % hmrcMongoVersion,
-    "org.typelevel"                %% "cats-core"                     % "2.10.0",
-    "com.beachape"                 %% "enumeratum-play-json"          % "1.6.3",
-    "com.googlecode.libphonenumber" % "libphonenumber"                % "8.13.29"
+    "uk.gov.hmrc"                  %% "play-frontend-hmrc-play-30"            % "8.5.0",
+    "uk.gov.hmrc"                  %% "play-conditional-form-mapping-play-30" % "2.0.0",
+    "uk.gov.hmrc"                  %% "bootstrap-frontend-play-30"            % bootstrapVersion,
+    "uk.gov.hmrc"                  %% "internal-auth-client-play-30"          % "2.0.0",
+    "uk.gov.hmrc.objectstore"      %% "object-store-client-play-30"           % "1.3.0",
+    "uk.gov.hmrc.mongo"            %% "hmrc-mongo-play-30"                    % hmrcMongoVersion,
+    "org.typelevel"                %% "cats-core"                             % "2.10.0",
+    "com.beachape"                 %% "enumeratum-play-json"                  % "1.6.3",
```

Comment:
Please bump to [1.8.0](https://github.com/lloydmeta/enumeratum/releases/tag/enumeratum-play-1.8.0), thanks.